app/controllers/uphold_connections_controller.rb

```diff
@@ -0,0 +1,59 @@
+class UpholdConnectionsController < ApplicationController
+  def login
+    # TODO: Make sure these params are safe
```

Comment:
what do you mean by safe?

Reply:
Just want to make sure the values won't consume Redis. Check is done in https://github.com/brave-intl/publishers/pull/2598/files#diff-5f2a026ba7311cca218cea5b67ee9cd2R5